tests/test_files/completion/import.skip.vv

```diff
@@ -0,0 +1,5 @@
+module main
```

Comment:
typicall we name it `_keep.vv`, so we could name this `_skip.vv`

Reply:
will change it immediately